### PR TITLE
Education section changes

### PR DESCRIPTION
**Pull request by VasaviLagishetty**
education
experience
mobile_view

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -145,35 +145,44 @@ syntax-highlight: True
...
 
-classes:
-  - image: "/img/timeline/spidertocat.png"
-    date: "August 2016 - Today"
-    description: "Engineering"
-  - image: "/img/timeline/baracktocat.jpg"
-    date: "August 2014 - March 2016"
-    description: "Intermediate"
-  - image: "/img/timeline/baracktocat.jpg"
+education:
```
Comment (SriSatyaLokesh) [inline _config.yml:161]: good one!

File: _config.yml [hunk 2]
```diff
@@ -229,7 +238,7 @@ keybase-in-footer: True
 
 pages_list:
   About: 'about'
-  Blog: 'latest-post'
+  #Blog: 'latest-post'
```
Comment (SriSatyaLokesh) [inline _config.yml:241]: i asked to do this

**Review by SriSatyaLokesh — APPROVED** [review]
Great debugging.
This is a great change that should get into master

**Review by puneethkanna — COMMENTED** [review]
Everything is fine, I think the education period must be from Btech to Schooling will be good, as we follow this in our resume pattern mostly.

**Comment by SriSatyaLokesh** [comment]
> Everything is fine, I think the education period must be from Btech to Schooling will be good, as we follow this in our resume pattern mostly.

Yes, I can explain and i'm confident that i can change your mind

**Review by puneethkanna — APPROVED** [review]

**Pull request merged**